File: Frontend/IMMRequest/src/app/models/fieldRange.ts
export class FieldRange {
    id: string;
    additionalFieldId: string;
    range: string;

    constructor(id: string, additionalFieldId: string, range: string){
        this.id = id;
        this.additionalFieldId = additionalFieldId;
        this.range = range;
    }
}

File: Backend/IMMRequest/IMMRequest.Domain/Entities/FieldRange.cs
using System;
using System.Collections.Generic;

namespace IMMRequest.Domain
{
    public class FieldRange
    {
        public Guid Id { get; set; }
        public string Range { get; set; }
        public AdditionalField AdditionalField { get; set; }
        public Guid AdditionalFieldId { get; set; }
        public FieldRange() { this.Id = Guid.NewGuid(); }
    }
}
